content/blog/2015-06-18-practical-anomaly-detection.md

```diff
@@ -83,8 +83,6 @@ ALERT InstanceLatencyOutlier
       >
         1
   FOR 30m
-  SUMMARY "{{$labels.instance}} in {{$labels.job}} is a latency outlier"
```

Comment:
They were only here previously because they were mandatory. This example is complicated enough without adding that too, and it's going to an automated system anyway.